platform/mbed_retarget.cpp

```diff
@@ -216,6 +233,13 @@ static FileHandle* default_console()
 #if DEVICE_SERIAL
 #  if MBED_CONF_PLATFORM_STDIO_BUFFERED_SERIAL
     static UARTSerial console(STDIO_UART_TX, STDIO_UART_RX, MBED_CONF_PLATFORM_STDIO_BAUD_RATE);
+#   if   CONSOLE_FLOWCONTROL == CONSOLE_FLOWCONTROL_RTS
+        console.set_flow_control(SerialBase::RTS, STDIO_UART_RTS, NC);
+#   elif CONSOLE_FLOWCONTROL == CONSOLE_FLOWCONTROL_CTS
+        console.set_flow_control(SerialBase::CTS, NC, STDIO_UART_CTS);
+#   elif CONSOLE_FLOWCONTROL == CONSOLE_FLOWCONTROL_RTSCTS
+        console.set_flow_control(SerialBase::RTSCTS, STDIO_UART_RTS, STDIO_UART_CTS);
```

Comment:
Was originally thinking this would use `concat(FlowControl, MBED_CONF_TARGET_CONSOLE_UART_FLOW_CONTROL)` and `SerialBase::MBED_CONF_TARGET_CONSOLE_UART_FLOW_CONTROL`, but yes, guess we should be spelling it out like this to not reference the CTS/RTS we're not using, so that's not necessary.